CONTRIBUTING.md

```diff
@@ -78,6 +79,24 @@ Signed-off-by: Jane Smith <jane.smith@email.com>
 ```
 You may type this line on your own when writing your commit messages. However, if your user.name and user.email are set in your git configs, you can use `-s` or `--signoff` to add the `Signed-off-by` line to the end of the commit message.
 
+## Changelog
+
+Flow framework maintains version specific changelog by enforcing a change to the ongoing [CHANGELOG](CHANGELOG.md) file adhering to the [Keep A Changelog](https://keepachangelog.com/en/1.0.0/) format.
```

Comment:
Nit 1: "Flow Framework".  I think. Feel free to disagree.
Nit 2: I think the version is now 1.1.0 (also link elsewhere).